adapter/src/index.js

```diff
@@ -1,3 +1,5 @@
+import { OfflineProvider } from '@dhis2/app-service-offline'
```

Comment:
In this file, an Offline Interface object is initialized, and an Offline Provider is added to the adapter